src/Zend/Registry.php

```diff
@@ -186,15 +186,4 @@ public function __construct($array = array(), $flags = parent::ARRAY_AS_PROPS)
     {
         parent::__construct($array, $flags);
     }
-
-    /**
-     * @param string $index
-     * @returns mixed
-     *
-     * Workaround for http://bugs.php.net/bug.php?id=40442 (ZF-960).
-     */
-    public function offsetExists($index)
```

Comment:
I removed this because the bug that is referenced here (http://bugs.php.net/bug.php?id=40442) has long since been fixed (it was only present in php 5.2.1: https://3v4l.org/VK1HQ).